src/Features/Core/Portable/Diagnostics/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -72,6 +75,9 @@ internal static class IDEDiagnosticIds
         public const string RemoveUnnecessaryParenthesesDiagnosticId = "IDE0047";
         public const string AddRequiredParenthesesDiagnosticId = "IDE0048";
 
+        public const string PreferBuiltInOrFrameworkTypeDiagnosticId = "IDE0049";
```

Comment:
Nit: extra empty line below

Reply:
I can clean in a subsequent PR :)  (i have a bunch that touch this file) i would prefer to not have to reset this one :D